src/vm/simplerwlock.hpp

```diff
@@ -148,7 +148,8 @@ class SimpleRWLock
         } CONTRACTL_END;
 
         m_RWLock = 0;
-        m_spinCount = (GetCurrentProcessCpuCount() == 1) ? 0 : 4000;
+        // Passing false here reduces ASP.NET Core Plaintext benchmark results from 1.2M to 0.8M RPS.
```

Comment:
Just curious - where (stacktrace) does the ASP.NET Core Plaintext takes this lock?